### PR TITLE
4.1 update python info

### PR DESCRIPTION
**Pull request by AlexicaWright**

### CODE REVIEW

**Review by gjmwoods — APPROVED** [review]

File: asciidoc/get-started.adoc [hunk 1]
```diff
@@ -56,7 +56,8 @@ In cases where at least one peer is below version 4.0, communication will occur
 [NOTE]
 Drivers 1.7 do not support multiple databases and Neo4j Fabric, features introduced in Neo4j 4.0.
 To be able to run multiple databases online concurrently and to do distributed queries over them, you must upgrade Drivers from 1.7 to 4.0.
-For information, see For information, see <<migration-guide#upgrade-driver, 4.0 Migration Guide -> Chapter 6. Upgrade Neo4j drivers>>.
+For information, see For information, see https://neo4j.com/docs/migration-guide/4.0/upgrade-driver/[4.0 Migration Guide -> Chapter 6. Upgrade Neo4j drivers].
```
Comment (davidoliverSP2) [inline asciidoc/get-started.adoc]:
```suggestion
For information, see https://neo4j.com/docs/migration-guide/4.0/upgrade-driver/[4.0 Migration Guide -> Chapter 6. Upgrade Neo4j drivers].
```

File: asciidoc/get-started.adoc [hunk 2]
```diff
@@ -24,15 +24,15 @@ The following languages and frameworks are officially supported by Neo4j:
 | .NET               | https://github.com/dotnet/standard/blob/master/docs/versions/netstandard2.0.md[.NET Standard 2.0]
 | Java               | Java 8+ (latest patch releases).
 | JavaScript         | All LTS versions of Node.JS, specifically the 4.x and 6.x series runtimes (https://github.com/nodejs/LTS).
+// | Python             | CPython 3.5 and above.
 |===
 
 [NOTE]
 --
-The 4.0 release of the Python and Go drivers are a work in progress. There is currently no release date available.
+The 4.1 release of the Go and Python drivers is a work in progress. There is currently no release date available.
```
Comment (davidoliverSP2) [inline asciidoc/get-started.adoc]:
```suggestion
The 4.1 Go and Python drivers are a work in progress. 
There is currently no release date available.
```

File: asciidoc/get-started.adoc [hunk 3]
```diff
@@ -24,15 +24,15 @@ The following languages and frameworks are officially supported by Neo4j:
...
 
-The 4.0 Python Driver is in alpha release stage with support for Python 3.5, 3.6, 3.7, 3.8, and this manual includes examples for using this alpha version.
-
-For more information on the 1.7 Driver documentation, including the 1.7 Go Driver, see {docs-base-uri}/driver-manual/1.7.
+For more information on the 1.7 Driver documentation, including the 1.7 and 1.8 Go Driver, see {docs-base-uri}/driver-manual/1.7.
+For more information on the 4.0 Python Driver, see {docs-base-uri}/driver-manual/4.0.
```
Comment (davidoliverSP2) [inline asciidoc/get-started.adoc]:
```suggestion
The latest Go Driver is 1.8.
For information on the 1.8 Go Driver, see {docs-base-uri}/driver-manual/1.7.

The latest Python Driver is 4.0.
For information on the 4.0 Python Driver, see {docs-base-uri}/driver-manual/4.0.
```

**Review by davidoliverSP2 — CHANGES_REQUESTED** [review]
Thanks @AlexicaWright 

A coupe of suggested changes for the information about versions.
I also spotted a typo elsewhere, and suggested a change.

**Review by davidoliverSP2 — APPROVED** [review]

**Pull request merged**